### PR TITLE
decorated mapbox with FreshTrak links

### PR DESCRIPTION
**Pull request by PhilNorman2**
Part of a POC for integration with the Mid-Ohio Food Bank [FreshTrak application](https://beta.freshtrak.com/).  Added FreshTrak links to Emergency Food mapbox when there is a match.  See [mofb-api PR #91](https://github.com/SCODEMeetup/mofb-api/pull/91) for details.

Example mapbox when there is a match with a FreshTrak agency id.:

 
Screen Shot 2020-10-21 at 5 39 47 PM

[FreshTrak Agency Link](https://beta.freshtrak.com/agency/events/6?utm_source=cbus_helper&utm_campaign=freshtrak_events)
[FreshTrak Zip Code Link](https://beta.freshtrak.com/events/list/43123?utm_source=cbus_helper&utm_campaign=freshtrak_events)

Example mapbox when there is no match with a FreshTrak agency id.

Screen Shot 2020-10-21 at 5 50 04 PM

### CODE REVIEW

**Review by vijayyadav06 — APPROVED** [review]

File: src/renderer/containers/map/infoBox/index.jsx [hunk 1]
```diff
@@ -26,6 +29,7 @@ export default class InfoBox extends React.Component {
 
   render() {
     const { details } = this.props;
+    console.log(`details: ${details}`);
```
Comment (vijayyadav06) [inline src/renderer/containers/map/infoBox/index.jsx:32]: leftover console.log please remove

**Review by vijayyadav06 — APPROVED** [review]

**Review by vijayyadav06 — APPROVED** [review]

File: src/renderer/containers/map/infoBox/index.jsx (hunk 1 above)
Comment (PhilNorman2) [reply src/renderer/containers/map/infoBox/index.jsx]: leftover console.log removed

File: src/renderer/containers/map/infoBox/index.jsx [hunk 2]
```diff
@@ -42,11 +46,17 @@ export default class InfoBox extends React.Component {
         ', '
       )}\n\nDataset: https://discovery.smartcolumbusos.com/dataset/handson_central_ohio/community_services_agencies`,
     };
+    const FRESHTRAK_UTM_PARAMS =
```
[the remark this answers is not in the document]
Comment (mihiramin89) [reply src/renderer/containers/map/infoBox/index.jsx:48]: @PhilNorman2 can you see this?

**Review by mihiramin89 — COMMENTED** [review]
added some comments.

File: src/renderer/containers/map/infoBox/index.jsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (PhilNorman2) [reply src/renderer/containers/map/infoBox/index.jsx:48]: Yes, I can see it now!

File: src/renderer/containers/map/infoBox/index.jsx [hunk 3]
```diff
@@ -65,6 +75,41 @@ export default class InfoBox extends React.Component {
...
+              <hr />
+              <h3>FreshTrak Food Resource Events</h3>
+            </div>
+          )}
+          {!isNil(details.freshtrakData) && !!details.freshtrakData.agencyURL && (
+            <div>
+              <p>
+                <a
+                  href={details.freshtrakData.agencyURL + FRESHTRAK_UTM_PARAMS}
+                  className="number"
```
[the remark this answers is not in the document]
Comment (PhilNorman2) [reply src/renderer/containers/map/infoBox/index.jsx:88]: I was just grabbing the style.  Is className="link" descriptive enough?

File: src/renderer/containers/map/infoBox/index.jsx [hunk 4]
```diff
@@ -65,6 +75,41 @@ export default class InfoBox extends React.Component {
...
+                  // eslint-disable-next-line react/jsx-no-target-blank
```
[the remark this answers is not in the document]
Comment (PhilNorman2) [reply src/renderer/containers/map/infoBox/index.jsx:89]: No, not needed anymore.  Had eslint version issues and didn't refactor afterwards.  Same is true for textnodes.

File: src/renderer/containers/map/infoBox/index.jsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (PhilNorman2) [reply src/renderer/containers/map/infoBox/index.jsx:88]: Wanted the same style as .number.  Creating .freshtrak-link with same style attributes.

File: src/renderer/containers/map/infoBox/infoBox.scss [hunk 5]
```diff
@@ -63,3 +63,11 @@
   text-align: center;
   padding-bottom: 2%;
 }
+
+.freshtrak-link {
```
Comment (mihiramin89) [inline src/renderer/containers/map/infoBox/infoBox.scss:67]: if these are all the same styles as the **.number** style above you can combine these styles with a comma like so `.number, .freshtrak-link {`

**Review by mihiramin89 — APPROVED** [review]

**Pull request merged**